### ISSUE TITLE
GetModel with includeManufacturer set to true doesn't return the manufacturer name

### ISSUE THREAD

**Issue by WilkoSki** [dannycabrera/GetDroidModel#2]
Using the following phone as an example:
Samsung,Galaxy J6,j6lte,SM-J600FN

The call to GetModel("unknown", true) will return "Galaxy J6" when the expected result is "Samsung Galaxy J6".

Looking in the GetModel method you have:

```
var deviceName = droidDevice.MarketingName == "" ? droidDevice.Model : droidDevice.MarketingName;

if (includeManufacturer && deviceName.StartsWith(droidDevice.Manufacturer, System.StringComparison.OrdinalIgnoreCase))
                    return $"{droidDevice.Manufacturer} {deviceName}";
                
return deviceName;
```

Seems like its only including the manufacturer if the device name starts with the manufacturer name.

Am I missing something here?

**Comment by dannycabrera** [comment]
@WilkoSki reviewing why I initially coded it this way and let you know shortly.

**Comment by dannycabrera** [comment]
Working on a fix as it should have been !deviceName.StartsWith(droidDevice.Manufacturer...

Thanks for letting me know @WilkoSki

**Comment by dannycabrera** [comment]
Fixed with commit 1af674b27113ac17c9f1977e5d05731218048a9a

**Comment by dannycabrera** [comment]
Fixed with commit a3f69d1bf52337ffb67fc132a603167d595462de

**Comment by dannycabrera** [comment]
Also submitted nuget package v1.4.0
